protokolle/2020-06-06_htw-pv3_postgresql_II.md

```diff
@@ -0,0 +1,139 @@
+# 2020-05-06 14:00 - 15:30 Wetterdatenverarbeitung mit PostgreSQL II -----#5
+
+#Auswertung Abgaben Aufgaben #1 und #2
+
+**Bewertungen**
```

Comment:
rather use `### Bewertungen`. The bold option `**` is better for inline text

Suggesting to change them in the whole document